### PR TITLE
Migrate to gazebo

### PR DESCRIPTION
**Pull request by Nicogene**
I managed to port the assignment to Gazebo

immagine

The problem for this assignment is that 5 points are given if in the center of the image there is the ball.
For checking this, it is compared to the HSV of the ball generated with the HSV of the central pixel of the image.

I have no issues for H and S, but the V(that depends on lighting and shadows) is giving me more than one headache.

immagine

I tried to move the sun in different places but the problem persists and gives problems like this one

immagine

My only idea is to remove the V in the check, in my opinion, H and S are sufficient. (see https://github.com/vvv-school/assignment_yarp-find-rgb/commit/442bf43a07dee516d57084ebf2f7fcd58084ba2e)

What do you think about it?

PS: On gitpod is working 

immagine

### CODE REVIEW

**Comment by Nicogene** [comment]
Here is how the lighting is handled by Gazebo

http://gazebosim.org/tutorials?tut=color_model

File: gazebo/models/assignment_yarp-find-rgb-world-interface/model.urdf [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  <link name="base_link"/>
+
+  <joint name="fake_base_fixed_joint" type="fixed">
+    <parent link="base_link"/>
+    <child link="base"/>
+  </joint>
+  
+  <link name="base">
+    <collision>
```
Comment (pattacini) [inline gazebo/models/assignment_yarp-find-rgb-world-interface/model.urdf:13]: Actually, you need neither the collision nor the inertial parts.

**Review by pattacini — APPROVED** [review]
Nicely done 👍🏻

File: gazebo/models/assignment_yarp-find-rgb-world-interface/model.urdf (hunk 1 above)
Comment (Nicogene) [reply gazebo/models/assignment_yarp-find-rgb-world-interface/model.urdf:13]: In theory not, but I don't remember why in practice you need a phisycal object(a cube) for using this plugin.

@traversaro do you know why?

BTW I set it very far, it should not pop out in the world

**Review by pattacini — DISMISSED** [review]
@Nicogene 
I forgot to apply this change: https://github.com/vvv-school/tutorial_gaze-interface/commit/c2b58b259a131ecc5c20a85b6034774d683b8aea.

So, please apply the same here.

**Pull request merged**